src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/security-recommendations/securityRecommendationsData.tsx

```diff
@@ -0,0 +1,155 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Since this is a function and is being used as such and not a React component I’d stick to the filename starting with a lowercase or do you all prefer it being uppercase? cc: @Vinnl

Reply:
That's the rule-of-thumb I've been using so far, but I don't have strong feelings one way or another.

Reply:
With components, I totally agree. In this case, I would not consider the file to contain a component in the classic sense?

Reply:
Yes exactly, I was trying to clarify that I was following the same rule-of-thumb as you were :)